Sources/Structures/Api/ApiRequestParameters.swift

```diff
@@ -22,14 +22,22 @@
 
 import Alamofire
 
+/**
+ *  Enum which keeps parameters type for request body
+ */
+public enum ParametersType {
+    case dictionary(_ parameter: Parameters?)
+    case array(_ array: [Parameters]?)
```

Comment:
Can we use [Any] or some more common type?

Reply:
why any?